src/frontend/apps/impress/src/pages/_app.tsx

```diff
@@ -16,15 +16,17 @@ type AppPropsWithLayout = AppProps & {
 export default function App({ Component, pageProps }: AppPropsWithLayout) {
   useSWRegister();
   const getLayout = Component.getLayout ?? ((page) => page);
-  const { t } = useTranslation();
+  const { t, i18n } = useTranslation();
 
   useEffect(() => {
+    document.documentElement.lang = i18n.language || 'fr';
```

Comment:
We don't need a fallback.
```suggestion
    document.documentElement.lang = i18n.language;
```